index.js

```diff
@@ -0,0 +1,190 @@
+                document.querySelector('#' + transription[i][j]).innerHTML = abc_rus[i][j];
+            }
+        }
+        leng = 'rus';
+    } else if (leng == 'rus') {
+        for (let i in transription) {
+            for (let j in transription[i]) {
+                document.querySelector('#' + transription[i][j]).innerHTML = abc[i][j];
+            }
+        }
+        leng = 'eng';  
+    }
+
+
+};
+
+function PressAnimation(event, catching) {
+    if (document.querySelector("#" + catching)) {
+        document.querySelector("#" + catching).classList.add("active");
+    }
+};
+
+
+window.onload = () => {
+    const info_message = document.createElement('p');
+    info_message.innerHTML = 'Сменить раскладку: левые Shift (сначала) + Ctrl. Сделано на Windows.';
+  
+    const textarea = document.createElement('textarea');
+    textarea.classList.add('text');
+    textarea.focus();
+    textarea.selectionStart = textarea.value.length;
+    textarea.value = "";
+
+    const keyboard = document.createElement('div');
+    keyboard.classList.add('keyboard');
+  
+    document.querySelector('body').append(info_message, textarea, keyboard);
+
+
+    function Actions(catching, keycode) {
+        switch (catching) {
+            case 'Backspace':
+                if (textarea.value) {
+                    textarea.value = textarea.value.slice(0, -1);
+                }
+                break;
+            case 'Tab':
+                textarea.value += '\t';
+                break;
+            case 'Enter':
+                textarea.value += '\n';
+                break;
+            case 'ControlLeft':
+                break;
+            case 'ControlRight':
+                break;
+            case 'AltLeft':
+                break;
+            case 'AltRight':
+                break;
+            case 'Delete':
+                break;
+            case 'ShiftLeft':
+                break; 
+            case 'ShiftRight':
+                break; 
+            case 'CapsLock':
+                break; 
+            case 'MetaLeft':
+                break; 
+            case 'ArrowUp':
+                textarea.value += '▲';
+                break;  
+            case 'ArrowDown':
+                textarea.value += '▼';
+                break; 
+            case 'ArrowLeft':
+                textarea.value += '◄';
+                break;
+            case 'ArrowRight':
+                textarea.value += '►';
+                break;  
+            
+            default: 
+                textarea.value += keycode;
+                break;   
+        }
+    
+    };
+
+
+
+    /*  creating keyboard  */   
+    for (let i in abc) {
+        let row = document.createElement('div');
+        row.classList.add('rows');
+        keyboard.append(row);
+        for (let j in abc[i]) {
+            let letter = document.createElement('span');
+            letter.classList.add('buttons');
+            letter.id += transription[i][j];

```

Comment:
[Data attributes](https://developer.mozilla.org/ru/docs/Web/Guide/HTML/Using_data_attributes) may be used to identify keys instead of ids.